packages/blocks/src/__snapshots__/index.spec.ts.snap

```diff
@@ -16,18 +23,20 @@ Array [
   "HomeBlocks",
   "ModalBlocks",
   "Button",
-  "CheckboxInputElement",
-  "Image",
```

Comment:
`Image` changing to being the block by default, with previous Image -> ImageElement requires major version bump.